### PR TITLE
Transition page for v1 users

### PR DESCRIPTION
**Pull request by jhkennedy**
This will become available at `hyp3-docs.asf.alaska.edu/v2-transition`

### CODE REVIEW

File: docs/v2-transition.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Welcome to HyP3 v2
+
+As of September 30, 2021, our beta HyP3 service available at <https://hyp3.asf.alaska.edu/>
+has been retired in favor of our new On Demand service powered by HyP3 version 2 (hereafter, just "HyP3").
+
+On Demand processing through HyP3 is now available directly in [Vertex](https://search.asf.alaska.edu/){target=_blank},
+ASF's data search portal. Vertex provides a friendly interface to request 
+processing jobs and review previous jobs. To learn how to request jobs through vertex, please see:
```
Comment (jhkennedy) [inline docs/v2-transition.md:8]: Not a big fan of this -- I really want to write *why* we retired v1 and *what benifit* the user is getting -- but I didn't like what I tried. Ideas?

**Review by hjkristenson — COMMENTED** [review]
Content looks reasonable. I'll leave the official review to those more familiar with URL forwarding structure. Will this page appear somewhere in hyp3-docs as well?

**Comment by jhkennedy** [comment]
> Content looks reasonable. I'll leave the official review to those more familiar with URL forwarding structure. Will this page appear somewhere in hyp3-docs as well?

The redirect stuff will send people to `hyp3-docs.asf.alaska.edu/v2-transition`, which will be this page

**Comment by hjkristenson** [comment]
> > Content looks reasonable. I'll leave the official review to those more familiar with URL forwarding structure. Will this page appear somewhere in hyp3-docs as well?
> 
> @hjkristenson The redirect stuff will send people to `hyp3-docs.asf.alaska.edu/v2-transition`, which will be this page

OK, so we're not planning to expose this on our site, just if people hit the old HyP3 website?

**Comment by jhkennedy** [comment]
Yes, I wasn't planning on putting it in our table of contents, but if you know the link, it'll show up as a page in our docs:
image

It will be discoverable via the search bar (and google) though

**Comment by hjkristenson** [comment]
> Yes, I wasn't planning on putting it in our table of contents, but if you know the link, it'll show up as a page in our docs:
> image
> 
> It will be discoverable via the search bar (and google) though

Ooh, thanks for that link @jhkennedy! I couldn't figure out how to view the content as it would appear on the web.

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**